docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+version: '2'
+
+services:
+  remacs:
+    build: .
+    image: remacs-build:latest
+    working_dir: ${PWD}
+    tty: true
+    # Unfortunately dumping doesn't work without this
```

Comment:
Yep, I was mostly linking it for future reference :)

Eventually I want Remacs to have a sensible dumping mechanism, at which point this will just work.